server/server.js

```diff
@@ -0,0 +1,93 @@
+var express = require('express');
+var app = express();
+var port = 3000;
+var cors = require('cors')
+var task = require("./task");
+app.use(cors());
+
+
+//express
+var bodyParser = require("body-parser");
+app.use(bodyParser.json());
+
+app.post("/todo", function(req, res, next) {
+  task.create({
+    title: req.body.title,
+    dueDate: req.body.dueDate,
+    address: req.body.address,
+    lat: req.body.lat,
+    lng: req.body.lng
+  }, function(err, task) {
+    if (err) {
+      res.status(500);
+      res.send(err);
+    } else {
+      res.status(201);
+      res.send(task);
+    }
+  });
+})
+app.delete("/todo/:id", function(req, res, next) {
+  var id = '' + req.params.id;
```

Comment:
This doesn't come as a string already? That kind of surprises me.

Reply:
You are right, that's a string already my bad :-)